sonic_platform_base/sonic_xcvr/api/xcvr_api.py

```diff
@@ -133,6 +133,23 @@ def get_transceiver_threshold_info(self):
         """
         raise NotImplementedError
 
+    def get_transceiver_status(self):
+        """
+        Retrieves transceiver status of this SFP
+        Returns:
+            A dict containing the keys/values
+        """
+        raise NotImplementedError
+
+    def get_transceiver_pm(self):
+        """
+        Retrieves PM for this xcvr
```

Comment:
could you add more details in this API what does PM mean and where its applicable?

Reply:
sure, added.